server/repository/src/db_diesel/changelog_row.rs

```diff
@@ -65,12 +65,11 @@ impl<'a> ChangelogRowRepository<'a> {
         ChangelogRowRepository { connection }
     }
 
-    /// # Arguments:
-    /// * earliest the first cursor to be included in the returned
-    /// * limit the number of entries to be returned
     pub fn changelogs(
         &self,
+        // Earliest the first cursor to be included in the returned
         earliest: u64,
+        // Limit the number of entries to be returned
```

Comment:
What is the proper way to document function arguments? I only found something as I did before with the markdown list...

It should have been 
"* `earliest`: the first cursor to be returned"
though...

Reply:
Your documentation was better then this in terms of rust docs. There was a suggestion with a question to change logs to this, I should have looked at the [guide](https://doc.rust-lang.org/rust-by-example/meta/doc.html) before making the change. 

New version: 45c00a876a925c2c9d597f913244ab80d23bde7e

![Screen Shot 2022-08-03 at 11 45 45 AM](https://user-images.githubusercontent.com/26194949/182497595-21d7d84e-a464-4f14-b8cf-1d67c00565e5.png)